### PR TITLE
docs: add patch to docs so to fix styling and code issues breaking the docs

### PR DESCRIPTION
**Pull request by Florence-Njeri**
**Description**
Fix styling for the generator tool docs and also breaking links on the website

**Related issue(s)**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: "Introduction"
...
+The AsyncAPI generator is a tool that generates anything you want using the **[AsyncAPI Document](generator/asyncapi-documents)** and **[Template](generator/template)** that are supplied as inputs to the AsyncAPI CLI. The generator was built with extensibility in mind; you can use the generator to generate anything you want, provided that it can be defined in a template, such as code, diagrams, markdown files, microservices, and applications. A number of [community-maintained templates](https://github.com/search?q=topic%3Aasyncapi+topic%3Agenerator+topic%3Atemplate) are now available for immediate usage.
```
Comment (derberg) [inline docs/index.md:6]: why `(generator/asyncapi-documents)` and not `(asyncapi-documents)` like with links in other documents?

I mean in some documents you make links with `generator/` prefix, and in some without it? is there a reason? All these documents display on the same level so logically for me they should follow the same pattern

File: docs/asyncapi-context.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: "AsyncAPI context"
+weight: 80
+---
+
+>This document is under construction.
```
Comment (derberg) [inline docs/asyncapi-context.md:6]: oh yeah, this one is missing

File: docs/index.md (hunk 1 above)
Comment (Florence-Njeri) [reply docs/index.md:6]: While doing manual testing on the website, the ones with the generator/ suffix were breaking without it.

**Review by derberg — APPROVED** [review]
LGTM 🚀

**Pull request merged**